Release 1.0 rc5 (#47)
kev-m/pyCoilGen: docs/source/configuration.md

## Generate Outputs

> The primary purpose of the application is to calculate the wire path of the coil that produces the desired target field.

Reviewer: ```suggestion
The primary purpose of `pyCoilGen` is to calculate the wire path of the coil that produces the desired target field.
```

### Generate Cylindrical PCB Output

> The application can optionally generate a PCB wire path that is suitable for wrapping around a cylinder.

Reviewer: ```suggestion
`pyCoilGen` can optionally generate a PCB wire path that is suitable for wrapping around a cylinder.
```

## Evaluate Results

> Once the application has calculated the wire path, it can also calculate some related values.

Reviewer: ```suggestion
Once `pyCoilGen` has calculated the wire path, it can also calculate some related values.
```

### Calculate Inductance

> The application uses [FastHenry2](https://www.fastfieldsolvers.com/software.htm) to calculate the inductance and resistance of the wire path.

Reviewer: ```suggestion
`pyCoilGen` uses [FastHenry2](https://www.fastfieldsolvers.com/software.htm) to calculate the inductance and resistance of the wire path.
```
